lib/modules/switch_board.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+import React from 'react-native';
+const { findNodeHandle } = React;
+const { ARSwitchBoardModule } = React.NativeModules;
+
+function presentNavigationViewController(component, route) {
```

Comment:
So if `presentNavigationViewController` is called on a react component, it will default to this implementation and call `ARSwitchBoardModule`, but in there there are `RCT_EXPORT_METHOD`s. Will it call into the default Eigen switchboard, or does it now have its own system?